src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -217,7 +211,23 @@ ov::Any CompiledModel::get_property(const std::string& name) const {
             return m_compiled_model_without_batch->get_property(name);
         } else if (name == ov::loaded_from_cache) {
             return m_compiled_model_without_batch->get_property(ov::loaded_from_cache.name());
+        } else if (name == ov::supported_properties) {
+            return std::vector<ov::PropertyName>{
+                ov::PropertyName{ov::optimal_number_of_infer_requests.name(), ov::PropertyMutability::RO},
+                ov::PropertyName{METRIC_KEY(SUPPORTED_METRICS), ov::PropertyMutability::RO},
+                ov::PropertyName{ov::model_name.name(), ov::PropertyMutability::RO},
+                ov::PropertyName{METRIC_KEY(SUPPORTED_CONFIG_KEYS), ov::PropertyMutability::RO},
+                ov::PropertyName{ov::execution_devices.name(), ov::PropertyMutability::RO},
+                ov::PropertyName{ov::auto_batch_timeout.name(), ov::PropertyMutability::RO}};
         } else {
+            // find config key among networks config keys
+            auto modelSupportedProperties =
+                m_compiled_model_without_batch->get_property(ov::supported_properties.name());
+            for (auto&& property : modelSupportedProperties.as<std::vector<ov::PropertyName>>()) {
+                if (property == name) {
+                    return m_compiled_model_without_batch->get_property(property);
+                }
+            }
```

Comment:
this pr does not solve timeout issue right? e.g
batch_compiled_model.set_property(auto_batch_timeout(200))
then : batch_compiled_model.get_property(auto_batch_timeout)
do you plan in a seperate PR?